integrationtest/mapper/pojo-base/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/mapping/definition/IndexedEmbeddedBaseIT.java

```diff
@@ -1302,6 +1306,148 @@ public Long fromDocumentIdentifier(String documentIdentifier,
 		}
 	}
 
+	@Test
+	public void customBridge_withParams_annotationMapping() {
```

Comment:
For consistency with the test methods above...

```suggestion
	public void includeEmbeddedObjectId_identifierBridge_withParams_annotationMapping() {
```